sdk/datalake-store/Azure.ResourceManager.DataLakeStore/api/Azure.ResourceManager.DataLakeStore.netstandard2.0.cs

```diff
@@ -53,193 +53,142 @@ public partial class DataLakeStoreAccountResource : Azure.ResourceManager.ArmRes
-        public string Name { get { throw null; } }
-        public Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType ResourceType { get { throw null; } }
-    }
-    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct CheckNameAvailabilityParametersType : System.IEquatable<Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType>
-    {
-        private readonly object _dummy;
-        private readonly int _dummyPrimitive;
-        public CheckNameAvailabilityParametersType(string value) { throw null; }
-        public static Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType MicrosoftDataLakeStoreAccounts { get { throw null; } }
-        public bool Equals(Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType other) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override bool Equals(object obj) { throw null; }
-        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
-        public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType left, Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType right) { throw null; }
-        public static implicit operator Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType (string value) { throw null; }
-        public static bool operator !=(Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType left, Azure.ResourceManager.DataLakeStore.Models.CheckNameAvailabilityParametersType right) { throw null; }
-        public override string ToString() { throw null; }
-    }
-    public partial class CreateFirewallRuleWithAccountParameters
-    {
-        public CreateFirewallRuleWithAccountParameters(string name, string startIPAddress, string endIPAddress) { }
-        public string EndIPAddress { get { throw null; } }
-        public string Name { get { throw null; } }
-        public string StartIPAddress { get { throw null; } }
-    }
-    public partial class CreateTrustedIdProviderWithAccountParameters
-    {
-        public CreateTrustedIdProviderWithAccountParameters(string name, string idProvider) { }
-        public string IdProvider { get { throw null; } }
-        public string Name { get { throw null; } }
-    }
-    public partial class CreateVirtualNetworkRuleWithAccountParameters
-    {
-        public CreateVirtualNetworkRuleWithAccountParameters(string name, string subnetId) { }
-        public string Name { get { throw null; } }
-        public string SubnetId { get { throw null; } }
-    }
     public partial class DataLakeStoreAccountBasic : Azure.ResourceManager.Models.ResourceData
```

Comment:
How about we rename this model to `DataLakeStoreAccountBasicData` as its base class is `ResourceData`?